sparksteps/steps.py

```diff
@@ -6,7 +6,7 @@
 
 REMOTE_DIR = '/home/hadoop/'
 S3_KEY_PREFIX = 'sparksteps/sources/'
-S3_URI_FMT = "s3://{bucket}/{key}"
+S3_PROTOCOL_PREFIX = 's3://'
```

Comment:
We should probably remove the module level variable and instead just hardcode `s3://` for better readability

Reply:
agreed